src/main/webapp/i18n/en/metis.json

```diff
@@ -4,6 +4,7 @@
             "noPosts": "There are no posts yet.",
             "addNewPost": "New post",
             "answerNow": "Reply now",
+            "startDiscussion": "Diskussion beginnen",
```

Comment:
This is the German translation :)

Reply:
I think I changed that already in 98a9a8a...that's outdated :)